common/script_values/00_men_at_arms_values.txt

```diff
@@ -244,36 +235,22 @@ culture_ai_weight_light_cavalry = {
 	if = { # Cultures that REALLY like light cavalry
 		limit = {
 			OR = {
-				culture_group = culture_group:iranian_group
-				
-				# Steppe cultures
-				culture_group = culture_group:turkic_group
-				culture_group = culture_group:mongolic_group
+				culture_group = culture_group:centaur_group
 			}
 		}
 		add = 80
 	}
 	if = { # Cultures that favours light cavalry
 		limit = {
 			OR = {
-				culture_group = culture_group:arabic_group
-				culture_group = culture_group:iranian_group
-				culture_group = culture_group:berber_group
-				culture_group = culture_group:iberian_group
-				culture_group = culture_group:west_slavic_group
-				culture_group = culture_group:indo_aryan_group
+				culture_group = culture_group:orc_group
 			}
 		}
 		add = 40
 	}
 	if = { # Cultures that dislike light cavalry
 		limit = {
 			OR = {
-				culture_group = culture_group:central_germanic_group
-				culture_group = culture_group:tibetan_group
-				culture_group = culture_group:baltic_group
-				culture_group = culture_group:burman_group
-				culture_group = culture_group:somalian_group
```

Comment:
Thinking about this, you should add a scripted trigger (`Warcraft-Guardians-of-Azeroth-2\common\scripted_triggers`) like `mountain_culture_trigger`. If you live in the mountains as dwarves do, you probably won't need heavy cavalry. Pikemen are a priority instead.

Reply:
what if you are a dwarf but your capital was in the mountains at the start of the game but has moved elsewhere?

Reply:
> what if you are a dwarf but your capital was in the mountains at the start of the game but has moved elsewhere?

You can mod it this way, but as you can see in the vanilla, the culture prefers units that fit the original region of the culture.